playbook/app/pb_kits/playbook/pb_dialog/child_kits/_dialog_footer.tsx

```diff
@@ -41,9 +38,9 @@ const DialogFooter = (props: DialogFooterProps) => {
 
   return (
     <>
-      <If condition={separator}>
-        <SectionSeparator />
-      </If>
+      {separator &&
+        <SectionSeparator aria={{dd: 'ff'}} className="dd" data={{dd: 'ff'}} id="d" text="ss"/>
```

Comment:
What are these attributes?

Reply:
Good catch, they were not supposed to be there, were used for a test only. thanks!